rocAL/rocAL_pybind/examples/rocAL_api_video_pipeline.py

```diff
@@ -152,17 +157,11 @@ def main():
                 print("**************ends*******************")
                 print("**************", i, "*******************")
         data_loader.reset()
-    #Your statements here
+    # Your statements here
     stop = timeit.default_timer()
 
     print('\n Time: ', stop - start)
 
 
 if __name__ == '__main__':
     main()
```

Comment:
Do we have EOF ?

Reply:
This issue is being observed multiple times. Each time I add EOF for this example, it shows as no EOF in file changes view.